### PR TITLE
[WOC] Create baseof template for bounty

### PR DESCRIPTION
**Pull request by chiptus**
##### Description
This PR introduces a baseof template to `/app/dashboard/templates/bounty` module.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

ui

##### Refers/Fixes

Refs: #194

### CODE REVIEW

**Comment by mbeacom** [comment]
:star: PR built and available in a preview environment **gitcoinco-web-pr-2377** [here](http://web.jx-gitcoinco-web-pr-2377.ci.gitcoin.co)

**Comment by mbeacom** [comment]
:star: PR built and available in a preview environment **gitcoinco-web-pr-2377** [here](http://web.jx-gitcoinco-web-pr-2377.ci.gitcoin.co)

**Comment by chiptus** [comment]
PR is ready for merge

File: app/dashboard/templates/bounty/new.html [hunk 1]
```diff
@@ -15,23 +17,15 @@
...
-  {% include 'shared/head.html' %}
+{% block head %}
   {% include 'shared/cards.html' %}
   <meta name="title" content="Create Funded Issue / Bounty | Gitcoin">
   <meta name="description" content="Instructions for creating an issue and posting a bounty to Gitcoin so developers across the world can discover your bounty and start working on your issue.">
   <link rel="stylesheet" href={% static "v2/css/submit_bounty.css" %}>
-</head>
+{% endblock head %}
 
-<body class="interior {{active}} g-font-muli" style="background-color: #FFFFFF;">
```
Comment (pinkiebell) [inline app/dashboard/templates/bounty/new.html]: Seems like the `style` attribute gone missing with this pr.
Or wasn't it used/or overruled anyway?

Comment (chiptus) [reply app/dashboard/templates/bounty/new.html]: it's not used. you can view https://gitcoin.co/funding/new and see that removing that doesn't change a thing

**Comment by mbeacom** [comment]
@chiptus Anything we can do to help with this?

cc @pinkiebell @gitcoinco/frontend

**Comment by chiptus** [comment]
I actually was waiting for a merge. I see there are conflicts now, I'll fix them and hope you'll merge it soon before other conflicts were introduced :D

**Comment by thelostone-mc** [comment]
@chiptus Any chance you are still on this ?

**Comment by chiptus** [comment]
I am, I'll merge the files today and fix the issues

**Comment by thelostone-mc** [comment]
@chiptus the codebase has changed a bit ! Are you still on this ? If so it might make to close this out and create a fresh PR

**Comment by chiptus** [comment]
Hey, sorry, really wanted to do this, but got a new computer and never got
to installing the environment and rebasing... If someone wants to take it,
go ahead 😁

**Pull request closed without merging**